cypress/helpers.ts

```diff
@@ -22,6 +22,11 @@ const password = Cypress.env('password') || 'admin';
 let isAuthEnabled = false;
 let authToken = null;
 let sessionToken = null;
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
[Issue](https://cdap.atlassian.net/browse/CDAP-19511) is causing cypress tests to fail. This is to force cypress to ignore it

Reply:
This issue is likely due to how we load/build files with webpack/node. Can probably fix this with the full angular conversion or a new webpack build.